File: app/src/main/java/com/pavlospt/rxfileexample/RxDemoApp.java
package com.pavlospt.rxfileexample;

import android.app.Application;
import timber.log.Timber;

public class RxDemoApp extends Application {

  @Override public void onCreate() {
    super.onCreate();
    if (BuildConfig.DEBUG) Timber.plant(new Timber.DebugTree());
  }
}

File: app/src/main/java/com/pavlospt/rxfileexample/RxDemoApp.kt
package com.pavlospt.rxfileexample

import android.app.Application
import timber.log.Timber
import timber.log.Timber.DebugTree

class RxDemoApp : Application() {
    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) Timber.plant(DebugTree())
    }
}